device/quanta/x86_64-quanta_ix8c_bwde-r0/Quanta-IX8C-56X/port_config.ini

```diff
@@ -0,0 +1,57 @@
+# name          lanes                 alias             index
+Ethernet1       60                    twentyfiveGigE1   0
```

Comment:
index start from 1, check the porting guide. it should match the front panel port number.

Reply:
index start from 0:
![start_0](https://user-images.githubusercontent.com/19216597/61513701-894a2280-aa30-11e9-9c1a-4fd739b5d54c.jpg)

index start from 1:
![start_1](https://user-images.githubusercontent.com/19216597/61513740-a1ba3d00-aa30-11e9-9bcf-277f38cece84.jpg)

Please advice. Thank you.